packages/client/utils/AtlassianManager.ts

```diff
@@ -831,4 +923,87 @@ export default abstract class AtlassianManager {
       `https://api.atlassian.com/ex/jira/${cloudId}/rest/api/3/screens/${screenId}/tabs/${tabId}/fields/${fieldId}`
     )
   }
+
+  async registerWebhooks(cloudId: string, webhooks: WebhookDetails[]) {
+    return this.post(`https://api.atlassian.com/ex/jira/${cloudId}/rest/api/3/webhook`, {
+      url: makeAppURL('http://localhost:3000', '/webhooks/jira'),
+      // url: makeAppURL(appOrigin, '/webhooks/jira'),
+      webhooks
+    })
+  }
+
+  async deleteWebhooks(cloudId: string, webhookIds: number[]) {
+    return this.delete(`https://api.atlassian.com/ex/jira/${cloudId}/rest/api/3/webhook`, {
+      webhookIds
+    })
+  }
+
+  async extendWebhooks(cloudId: string, webhookIds: number[]) {
+    return this.put(`https://api.atlassian.com/ex/jira/${cloudId}/rest/api/3/webhook/refresh`, {
+      webhookIds
+    })
+  }
+
+  async getWebhooks(cloudId: string) {
+    return this.get<JiraPageBean<Webhook>>(
+      `https://api.atlassian.com/ex/jira/${cloudId}/rest/api/3/webhook`
+    )
+  }
+  async getAllWebhooks(cloudIds: string[]) {
+    return Promise.all(cloudIds.map((cloudId) => this.getWebhooks(cloudId)))
```

Comment:
+1 if the return value is extended with `cloudId` then the parent function wouldn't need to dangerously index into the `cloudIds` array